app/models/experimentation/notifications_experiment.rb

```diff
@@ -39,22 +41,45 @@ def self.eligible_patients
                                              .having("count(patient_id) > 1"))
     end
 
-    def enroll_patients(date)
+    def enroll_patients(date, limit = MAX_PATIENTS_PER_DAY)
       eligible_patients(date)
-        .limit(MAX_PATIENTS_PER_DAY)
+        .limit([remaining_enrollments_allowed(date), limit].min)
         .includes(:assigned_facility, :registration_facility, :medical_history)
         .includes(latest_scheduled_appointments: [:facility, :creation_facility])
-        .in_batches(of: ENROLLMENT_BATCH_SIZE)
+        .in_batches(of: MEMBERSHIPS_BATCH_SIZE)
         .each_record { |patient| random_treatment_group.enroll(patient, reporting_data(patient, date)) }
```

Comment:
Do we want to log how many patients were enrolled? Or log anything else that comes out of this?